jetcd-core/src/main/java/io/etcd/jetcd/LeaseImpl.java

```diff
@@ -105,6 +105,12 @@
     );
   }
 
+  @Override
+  public CompletableFuture<LeaseGrantResponse> grant(long ttl, long timeout, TimeUnit unit) {
+    return connectionManager.execute(() -> this.stub.withDeadlineAfter(timeout,unit).leaseGrant(LeaseGrantRequest
+            .newBuilder().setTTL(ttl).build()),timeout,unit,LeaseGrantResponse::new);
```

Comment:
here the same timeout is applied to two different concept which is quite confusing, would be better to use only one of the two

Reply:
um, I use as  a grpc invocation timeout then